content/docs/tasks/traffic-management/circuit-breaking/index.md

```diff
@@ -1,39 +1,51 @@
 ---
 title: Circuit Breaking
-description: This task demonstrates the circuit-breaking capability for resilient applications
+description: This task shows you how to configure circuit breaking for connections, requests, and outlier detection.
 weight: 50
 keywords: [traffic-management,circuit-breaking]
 ---
 
 > This task uses the new [v1alpha3 traffic management API](/blog/2018/v1alpha3-routing/). The old API has been deprecated and will be removed in the next Istio release. If you need to use the old version, follow the docs [here](https://archive.istio.io/v0.7/docs/tasks/traffic-management/).
 
-This task demonstrates the circuit-breaking capability for resilient applications. Circuit breaking allows developers to write applications that limit the impact of failures, latency spikes, and other undesirable effects of network peculiarities. This task will show how to configure circuit breaking for connections, requests, and outlier detection.
+This task shows you how to configure circuit breaking for connections, requests,
+and outlier detection.
 
 ## Before you begin
 
 * Setup Istio by following the instructions in the
   [Installation guide](/docs/setup/).
 
-*   Start the [httpbin]({{< github_tree >}}/samples/httpbin) sample
-    which will be used as the backend service for our task.
+*   Add the [httpbin]({{< github_tree >}}/samples/httpbin) sample to the mesh:
 
-    If you have enabled [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection), do
+    If you have enabled [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection), run this command:
```

Comment:
automatic injection is the default and what we are leading with.  I'd suggest removing the text around manual injection and the conditional in this text as it may cause confusion.